contributing/TESTING.md

```diff
@@ -113,6 +113,10 @@ Command                                                                 | Descri
 `gulp firebase --file path/to/file`                                     | Same as above, but copies over the file specified as `firebase/index.html`.
 `gulp firebase --min`                                                   | Same as `gulp firebase`, but uses minified files of the form `/dist/v0/amp-component-name.js` instead of unminified files of the form `/dist/v0/amp-component-name.max.js`.
 `gulp firebase --nobuild`                                               | Same as `gulp firebase`, but skips the `gulp build` step.
+`gulp e2e`                                                              | Runs all end to end tests on Chrome.
```

Comment:
Style nit: Perhaps replace `end to end` with `end-to-end` everywhere?

Reply:
Done, makes it easier to read.